lib/index.d.ts

```diff
@@ -1,6 +1,8 @@
-export * from './followStore';
-export * from './Store';
-export * from './StorePersistentLocalStorageDriver';
-export * from './useStore';
-export { StoreEvent, StoreEventType } from './StoreEvent';
-export { StorePersistentDriver } from './StorePersistentDriver';
+export * from './followStore';
+export * from './Store';
+export * from './StorePersistentLocalStorageDriver';
+export * from './useStore';
+export * from './useIsolatedStore';
+export { StoreEvent, StoreEventType } from './StoreEvent';
+export { StorePersistentDriver } from './StorePersistentDriver';
+export * from './compare';

```

Comment:
How do you plan to use it outside of the module?

Reply:
We exported `areSimilar` function from `./compare`  for lazy developers to equality prev and next states. In [README](https://github.com/ibitcy/react-stores/pull/31/files#diff-04c6e90faac2675aa89e2176d2eec7d8R569) we mention this.

`./useIsolatedStore` exports a new hook.

`./StorePersistentDriver` exports interfaces and abstractions for creating drivers for persistent stores

`StoreEventType` also need for  using `on(StoreEventType.Something, ...`

Maybe only `StoreEvent` exporting is not necessary  🤷🏽

Reply:
`StoreEvent` is really unnecessary, but one of our teammates asked me to export it. So we hold it for a while.